man/man1/pkcshsm_mk_change.1.in

```diff
@@ -19,17 +19,253 @@ for a concurrent HSM master key change for openCryptoki.
 .br
 
 .SH DESCRIPTION
-tbd
+Manages and controls the re-enciphering of secure keys for a concurrent HSM
+master key change for openCryptoki. Secure keys are encrypted by the HSM master
```

Comment:
for CCA there are 4 different types of master key, so I would write:
Secure keys are encrypted by HSM master key(s).

Reply:
fixed